local_server/qwen3_tts_server/local_server.py

```
@@ -0,0 +1,261 @@
+                if os.path.exists(ref_wav):
+                    with torch.no_grad():
+                        self.cached_prompt = self.model.create_voice_clone_prompt(
+                            ref_audio=ref_wav,
+                            ref_text=self.ref_text
+                        )
+                    torch.save(self.cached_prompt, self.pt_path)
+                    logger.info("✅ 音色提取并保存完成")
+                else:
+                    logger.error(f"❌ 找不到参考音频: {ref_wav}")
+
+            logger.info(f"🚀 语音引擎初始化完成，总耗时: {time.time() - t0:.2f}s")
```

Ruff (0.14.14)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)